README.md

```
@@ -2,27 +1,237 @@
+![Next.js](https://img.shields.io/badge/Next.js-black?style=flat-square&logo=next.js)
+![shadcn/ui](https://img.shields.io/badge/shadcn/ui-black?style=flat-square&logo=shadcnui)
 
-## Mentor
-- mahbub23
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Learnathon-By-Geeky-Solutions_procastinators&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Learnathon-By-Geeky-Solutions_procastinators)
+[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=Learnathon-By-Geeky-Solutions_procastinators&metric=ncloc)](https://sonarcloud.io/summary/new_code?id=Learnathon-By-Geeky-Solutions_procastinators)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=Learnathon-By-Geeky-Solutions_procastinators&metric=coverage)](https://sonarcloud.io/summary/new_code?id=Learnathon-By-Geeky-Solutions_procastinators)
+
+# Finance Tracker
+
+###### Simplify Finances, Ease Collaboration
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h6

(MD001, heading-increment)